source/driverAtmelRFInterface.cpp

```diff
@@ -878,6 +878,30 @@ void rf_if_set_channel_register(uint8_t channel)
 }
 
 /*
+ * \brief Function enables RF irq pin interrupts in RF interface.

```

Comment:
Should we then change them to entire file? Or all source files? Not just these two new functions?